scripts/check-events

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/ruby
+# coding: utf-8
+
+require 'find'
+
+COMMON_EVENTS = ['EventsReceived', 'EventsSent']
+# Sources implicitly emit EventsSent in the common pipeline code
+SOURCE_EVENTS = ['BytesReceived'] + COMMON_EVENTS - ['EventsSent']
+# Sinks implicitly emit EventsReceived in the common pipeline code
+SINK_EVENTS = COMMON_EVENTS + ['BytesSent'] - ['EventsReceived']
+# Transforms implicitly emit the common events in the topology, and have
+# no other emit requirements.
+TRANSFORM_EVENTS = []
```

Comment:
Are these used anywhere? I'm not seeing it.

Reply:
Oh, oops, those were part of the old script that I didn't purge out. Fixed in f2e1a49c4